source/blog/2018-08-31-the-ember-times-issue-62.md

```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification with Ember Addons](https://github.com/emberjs/rfcs/pull/367) 🎁
+[Module Unification Packages](https://github.com/emberjs/rfcs/pull/367) is a new RFC, created by [@mixonic](https://github.com/mixonic), that sets out to describe how packages, **Ember Addons**, work and how apps and addons will migrate to the new [Module Unification](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md) structure from the classic structure. This RFC iterates on and is set to replace another RFC called [Module Unification Namespaces](https://github.com/emberjs/rfcs/pull/309) which had some syntax, like the `::` syntax, that proved problematic.
+
+This RFC proposes to add a new `{{use}}` helper that will be used to call external components that are provided by addons. This helper is a subset of the javascript imports that we are used to. 
+
+An example: In this template the `{{use}}` helper imports a component `Widget` from the `gadget` addon.
+
+```hbs
+{{! invokes node_modules/gadget/src/ui/components/Widget/component.js }}
+
+{{use Widget from 'gadget'}}
+<Widget @options=someOptions @value=someValue />
```

Comment:
My syntax here was incorrect, I need to fix it in the RFC. This should be:

```hbs
{{! invokes node_modules/gadget/src/ui/components/Widget/component.js }}

{{use Widget from 'gadget'}}
<Widget @options={{someOptions}} @value={{someValue}} />
```

Reply:
Oh good catch!